[DOCS] : Improved mimi model card (#39824)
huggingface/transformers: docs/source/en/model_doc/mimi.md

-->

<div style="float: right;">
    <div class="flex flex-wrap space-x-1">
        <img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-DE3412?style=flat&logo=pytorch&logoColor=white">
        <img alt="FlashAttention" src="https://img.shields.io/badge/%E2%9A%A1%EF%B8%8E%20FlashAttention-eae0c8?style=flat">
        <img alt="SDPA" src="https://img.shields.io/badge/SDPA-DE3412?style=flat&logo=pytorch&logoColor=white">
    </div>
</div>

# Mimi

> [Mimi](https://kyutai.org/Moshi.pdf) is a neural audio codec model with pretrained and quantized variants, designed for efficient speech representation and compression. The model operates at 1.1 kbps with a 12 Hz frame rate and uses a convolutional encoder-decoder architecture combined with a residual vector quantizer of 16 codebooks. Mimi outputs dual token streams i.e. semantic and acoustic to balance linguistic richness with high fidelity reconstruction. Key features include a causal streaming encoder for low-latency use, dual-path tokenization for flexible downstream generation, and integration readiness with large speech models like Moshi.

Reviewer: ```suggestion
[Mimi](https://huggingface.co/papers/2410.00037) is a neural audio codec model with pretrained and quantized variants, designed for efficient speech representation and compression. The model operates at 1.1 kbps with a 12 Hz frame rate and uses a convolutional encoder-decoder architecture combined with a residual vector quantizer of 16 codebooks. Mimi outputs dual token streams i.e. semantic and acoustic to balance linguistic richness with high fidelity reconstruction. Key features include a causal streaming encoder for low-latency use, dual-path tokenization for flexible downstream generation, and integration readiness with large speech models like Moshi.
```

> Its architecture is based on [Encodec](model_doc/encodec) with several major differences:

Reviewer: You can remove all this text here and replace with:

```md
You can find the original Mimi checkpoints under the [Kyutai](https://huggingface.co/kyutai/models?search=mimi) organization.
```

* it uses a much lower frame-rate.
* it uses additional transformers for encoding and decoding for better latent contextualization
* it uses a different quantization scheme: one codebook is dedicated to semantic projection.

>[!TIP]

> >This model was contributed by [Yoach Lacombe (ylacombe)](https://huggingface.co/ylacombe).
> >The original code can be found [here](https://github.com/kyutai-labs/moshi).
> >Click on the Mimi models in the right sidebar for more examples of how to apply Mimi to encode and decode an audio using this model.

Reviewer: ```suggestion
> This model was contributed by [ylacombe](https://huggingface.co/ylacombe).
>
> Click on the Mimi models in the right sidebar for more examples of how to apply Mimi to encode and decode audio.
```

> The example below demonstrates how to encode and decode an audio with [`Pipeline`] or the [`AutoModel`] class.

Reviewer: ```suggestion
The example below demonstrates how to encode and decode audio with the [`AutoModel`] class.
```

> ```python 

Reviewer: Wrap this code snippet with:

```md
<hfoptions id="usage">
<hfoption id="AutoModel">

...

</hfoption>
</hfoptions>
```
